Zend/tests/bug81076.phpt

```diff
@@ -0,0 +1,12 @@
+--TEST--
+Bug #81076 Invalid implicit binds cause incorrect static var count in closure debug info
+--FILE--
+<?php
+var_dump(fn() => [$why, $do, $we, $count]);
+?>
+--EXPECT--
+object(Closure)#1 (1) {
+  ["static"]=>
```

Comment:
Do we want this empty array ? 

May be better to add conditionally to debug info based on actual count ?

Reply:
No, we don't.